sdk/eventhub/Azure.Messaging.EventHubs/stress/src/Program.cs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Producer;
+
+namespace Azure.Messaging.EventHubs.Stress
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            var testsToRun = ParseArguments(args);
+
+            // Determine which tests to run
+            testsToRun.TryGetValue("EventProd", out var runEventProducerTest);
```

Comment:
How does this work with the number of pods?   I don't think we want to allow concurrent tests in the same pod.   We should be able to run concurrent tests in different pods... but that would likely be outside of the purview of this executable.

Reply:
The changes here do concurrent tests in different pods. The [values.yaml](https://github.com/Azure/azure-sdk-for-net/blob/52bfa862fa9cdae6172aa0462a465bbe862a3c34/sdk/eventhub/Azure.Messaging.EventHubs/stress/values.yaml) config has a `scenarios` key. When the test is deployed, it will be replicated for each scenario in that list, with the scenario value passed down through the configs. The scenario name is then passed to the app here: https://github.com/Azure/azure-sdk-for-net/blob/52bfa862fa9cdae6172aa0462a465bbe862a3c34/sdk/eventhub/Azure.Messaging.EventHubs/stress/templates/deploy-job.yaml#L14-L15